test/browser/suspense.test.js

```diff
@@ -0,0 +1,354 @@
+/*eslint-env browser, mocha */
+/** @jsx h */
+import { setupRerender } from 'preact/test-utils';
+import { createElement as h, render, Component, Suspense, lazy, Fragment } from '../../src/index';
+import { setupScratch, teardown } from '../_util/helpers';
+
+class LazyComp extends Component {
+	render() {
+		return <div>Hello from LazyComp</div>;
+	}
+}
+
+function CustomSuspense({ isDone, start, name }) {
+	if (!isDone()) {
+		throw start();
+	}
+
+	return (
+		<div>
+			Hello from CustomSuspense {name}
+		</div>
+	);
+}
+
+class Catcher extends Component {
+	constructor(props) {
+		super(props);
+		this.state = { error: false };
+	}
+
+	componentDidCatch(e) {
+		this.setState({ error: e });
+	}
+
+	render(props, state) {
+		return state.error ? <div>Catcher did catch: {state.error.message}</div> : props.children;
+	}
+}
+
+function createSuspension(name, timeout, t) {
```

Comment:
What's `t`? Can you give it a more legible name? Perhaps `error`?

Reply:
Done